plugins/lang/python/python_loader.cc

```diff
@@ -517,7 +517,10 @@ class mldb_wrapper(object):
     def wrap(mldb):
         return Wrapped(mldb)
 
-    class ResponseException(Exception):
+    class MldbBaseException(Exception):
```

Comment:
Will be applied to PR TooManyRedirectsException from https://github.com/mldbai/mldb/pull/155/files